docs/release_notes/pending_release.rst

```diff
@@ -12,6 +12,8 @@ Documentation
 
 * Added text discussing black box methods to ``introduction.rst``.
 
+* Added a section to ``introduction.rst`` that describes the links between sgit aliency algorithms and implementations.
```

Comment:
typo here, I think you mean this?

```suggestion
* Added a section to ``introduction.rst`` that describes the links between saliency algorithms and implementations.
```

Reply:
I think when I try to search on text strings I inadvertently end up typing in the editing panel <ugh>